### DISCUSSION TITLE
Domain/DNS management

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#32] [Ideas]
Foundations _usually_ end up holding resources like domains (that's the long-term benefit of the neutral party), however.. that can also introduce a bottleneck if a project is used to managing their own DNS records for docs, builds, website, blogs, etc.

I use Namecheap for most of my domains, and they have a mechanism to delegate access for domain management: https://www.namecheap.com/support/knowledgebase/article.aspx/192/46/how-do-i-share-access-to-my-domain-with-other-users/  (Foundation can own domain, project can still manage).

- Is this good (enough)? 
- Other providers that do this?
- Concerns?

**Comment by Sanne** [comment]
That looks reasonable, I just hope it will be flexible enough:

DNS management has become even more complex in recent times; for example for hibernate.org beyond various websites we also have associated email addresses for members, and gmail recently required us to reconfigure fairly advanced details to make sure our emails wouldn't be flagged as suspicious. It's honestly going beyond my ability to understand DNS, this was hard to fix even for a specialist.

I guess I'm warning against needing to be using a specific platform, should it not work well for some project we should allow some pragmatic plan B.

**Reply by ebullient** [reply]
Agreed. What I guess I am asserting here is a preference for providers that allow delegation.. some way for projects to manage their own things without having to share credentials. 

The Foundation must own the domain, but I'd strongly prefer having ways for projects to do what they need to do with it. Namecheap does that delegation, and forwardemail has an API for managing mailing addresses and has the DNS config that makes google happy.

**Reply by Sanne** [reply]
Perfect, agreed.

**Comment by ebullient** [comment]
A Commonhaus account has been created at NameCheap, and all of the Commonhaus Foundation domains have been moved there.

I can confirm that management access can be delegated to another account (i.e. I can still manage those domains using my own NameCheap account, while NameCheap is the domain owner.
